cumulusci/tasks/metadata/package.py

```diff
@@ -249,6 +276,27 @@ def check_delete_excludes(self, item):
     def _parse_subitem(self, item, subitem):
         return [item + "/" + self.strip_extension(subitem)]
 
+    def _strip_component(self,item,component_list):
+        path = self.directory + "/" + item
+        # Check if it is a directory and take action
+        if os.path.isdir(path):
+            if os.path.exists(path+"-meta.xml"):
+                if item not in component_list:
```

Comment:
Could condense here into 
```suggestion
        if os.path.isdir(path) and os.path.exists(path+"-meta.xml") and item not in component_list:
```